app/forms/idv/doc_pii_form.rb

```diff
@@ -45,6 +46,8 @@ def validate_pii
         errors.add(:pii, dob_error, type: :dob_error)
       elsif !dob_meets_min_age?
         errors.add(:pii, dob_min_age_error, type: :dob_min_age_error)
+      elsif address1.blank?
```

Comment:
Could we use [ActiveRecord validation helpers](https://guides.rubyonrails.org/active_record_validations.html) for this?

```rb
validates :address1, presence: { message: proc { I18n.t('doc_auth.errors.alerts.address_check') } }
```

Related Slack discussion: https://gsa-tts.slack.com/archives/C0NGESUN5/p1692037164572949

Reply:
Thanks for the suggestion, I tried that and our tests revealed that that approach produces slightly different errors:
```
# From the current :validate_pii approach
=> {:pii=>["We couldn’t read the address on your ID. Try taking new pictures."]}
# From an ActiveRecord Validation
=> {:address1=>["We couldn’t read the address on your ID. Try taking new pictures."]}
```

I broke that work out into a separate ticket here: https://cm-jira.usa.gov/browse/LG-10760 rather than address it as part of this PR for a few reasons:
- These validations are quite important for user success.
- I have a suspicion that doing this may require transitioning all the validations out of `:validate_pii`.

Reply:
looks like address and names etc considered as one validation entity pii.